modules/liquid.nix

```diff
@@ -189,17 +189,17 @@ in {
         if ! test -e ${cfg.dataDir}; then
           mkdir -m 0770 -p '${cfg.dataDir}'
         fi
-        cp '${configFile}' '${cfg.dataDir}/liquid.conf'
-        chmod o-rw  '${cfg.dataDir}/liquid.conf'
+        cp '${configFile}' '${cfg.dataDir}/elements.conf'
+        chmod o-rw  '${cfg.dataDir}/elements.conf'
         chown -R '${cfg.user}:${cfg.group}' '${cfg.dataDir}'
-        echo "rpcpassword=$(cat /secrets/liquid-rpcpassword)" >> '${cfg.dataDir}/liquid.conf'
-        echo "mainchainrpcpassword=$(cat /secrets/bitcoin-rpcpassword)" >> '${cfg.dataDir}/liquid.conf'
+        echo "rpcpassword=$(cat /secrets/liquid-rpcpassword)" >> '${cfg.dataDir}/elements.conf'
+        echo "mainchainrpcpassword=$(cat /secrets/bitcoin-rpcpassword)" >> '${cfg.dataDir}/elements.conf'
       '';
       serviceConfig = {
         Type = "simple";
         User = "${cfg.user}";
         Group = "${cfg.group}";
-        ExecStart = "${pkgs.liquidd}/bin/liquidd ${cmdlineOptions}";
+        ExecStart = "${pkgs.elementsd}/bin/elementsd ${cmdlineOptions}";
```

Comment:
Don't we need to specify `-chain=liquidv1` in the cmdlineOptions?

Reply:
fixed